testing/routes.md

```diff
@@ -0,0 +1,129 @@
+We can test server routes using [Supertest](https://github.com/visionmedia/supertest) and [tape](https://github.com/substack/tape).
+
+
+### Setting up
+
+We do this first and foremost by _exporting_ the Express `app` object from `server.js` (or wherever it happens to be defined):
+
+```
```

Comment:
Maybe put `// server.js` here so scanning the code snippets (without reading the preceding paragraph) makes sense?